window/src/os/windows/event.rs

```diff
@@ -11,29 +9,26 @@ unsafe impl Sync for EventHandle {}
 impl Drop for EventHandle {
     fn drop(&mut self) {
         unsafe {
-            CloseHandle(self.0);
+            CloseHandle(self.0).unwrap();
         }
     }
 }
 
 impl EventHandle {
     pub fn new_manual_reset() -> anyhow::Result<Self> {
-        let handle = unsafe { CreateEventW(null_mut(), 1, 0, null()) };
-        if handle.is_null() {
-            return Err(IoError::last_os_error().into());
-        }
+        let handle = unsafe { CreateEventW(None, true, false, PCWSTR::null()) }?;
         Ok(Self(handle))
     }
 
     pub fn set_event(&self) {
         unsafe {
-            SetEvent(self.0);
+            SetEvent(self.0).unwrap();
```

Comment:
I try really hard to avoid bare `.unwrap()`.  Suggested alternatives:

* If the error is "totally impossible" and "can never happen", use `.expect("reason this can never happen")` so that the code documents why it is impossible
* If the error is possible at runtime and we just don't care, consider using `.ok()` instead of `.unwrap()` to discard the `must_use` attribute
* If the error is possible at runtime and something we might need to handle, change the signature of the calling function to some kind of `Result` and allow the caller to decide whether and how to handle it